Dockerfile

```diff
@@ -1,16 +1,16 @@
-FROM fedora:32
+FROM quay.io/centos/centos:stream
 
 env LIBGUESTFS_BACKEND direct
 
-RUN dnf update -y && dnf install --setopt=install_weak_deps=False -y \
+RUN yum update -y && yum install --setopt=install_weak_deps=False -y \
     libguestfs  \
     libguestfs-devel \
```

Comment:
This will result in the version of libguestfs (and its dependencies) that are available in CentOS Stream 8 being installed. For KubeVirt, I'm planning to use the packages that are in Advanced Virtualization instead.

The difference can be quite dramatic: in the case of libguestfs, you'd go from 1.40.2-28 to 1.44.0-3.

So what you should do is install the `centos-release-advanced-virtualization` package first, and only then installing the libguestfs packages to make sure you get the ones from AV.

Reply:
thanks! I changed the PR accordingly